database/communication.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"context"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const getCommunications string = "Select id, name, subject, frequency_throttle, template from membership.communication;"
+const getCommunication string = "Select id, name, subject, frequency_throttle, template from membership.communication where name = $1;"
+const getLastCommunication string = "Select created_at from membership.communication_log where member_id = $1 and communication_id = $2;"
+const logCommunication string = "Insert into membership.communication_log (member_id, communication_Id) values ($1, $2);"
+
+// Communication defines an email communication
+type Communication struct {
+	ID                int    `json:"id"`
+	Name              string `json:"name"`
+	Subject           string `json:"subject"`
+	FrequencyThrottle int    `json:"frequency_throttle"`
```

Comment:
Would it be alright if you could change `frequency_throttle` to `frequencyThrottle`? A lot of our models are camelCase. Want to make sure we are consistent.